tests/specs/file.js

```diff
@@ -536,40 +819,38 @@ define(["dist/amd-resolver"], function(Resolver) {
     });
 
 
-    describe("hasProtocol", function() {
-      it("Path with forward slash", function() {
+    describe("when calling hasProtocol", function() {
+      it("a path with a forward slash should have a protocol", function() {
```

Comment:
I think we can be a bit more assertive with the wording, so maybe changing the `should have` with a `has`